lib/utils/compiler.js

```diff
@@ -41,19 +41,22 @@ function setUpHookForCompiler(compiler, outputOptions, options) {
     });
 
     if (outputOptions.infoVerbosity === 'verbose') {
+        const resolveCompilationName = compilation => {
```

Comment:
Why not make use of the inline approach?

```js
        const resolveCompilationName = compilation => 
          compilation.name ? compilation.name : '';
```

Reply:
:thinking:  IMO this is much more inferable.

Reply:
Just personal preference :smile: